cmd/livepeer/livepeer.go

```diff
@@ -96,7 +96,7 @@ func main() {
 	testnet := flag.Bool("testnet", false, "Set to true to connect to testnet")
 	rinkeby := flag.Bool("rinkeby", false, "Set to true to connect to rinkeby")
 	controllerAddr := flag.String("controllerAddr", "", "Protocol smart contract address")
-	gasLimit := flag.Int("gasLimit", 4000000, "Gas limit for ETH transactions")
+	gasLimit := flag.Int("gasLimit", 0, "Gas limit for ETH transactions")
```

Comment:
Small addition - defaulting to 0 here will force all transactions created to use the gas estimator to set the gas limit on transactions such that the gas limit = estimate + 10%